.github/workflows/ci.yml

```
@@ -55,6 +55,7 @@ jobs:
           - 'basic'
           - 'configure'
           - 'azuread'
+          - 'config_plugins'
```

Test-kitchen translates `_` to `-` which is why this is failing.
```suggestion
          - 'config-plugins'
```